PollBuddy-Server/frontend/src/pages/PollHistory/PollHistory.js

```diff
@@ -0,0 +1,89 @@
+import React, {Component} from "react";
+import {Link, Redirect} from "react-router-dom";
+import {MDBContainer} from "mdbreact";
+
+//TODO This page can only be access when one is logged in
+//TODO Need to dynamically grab what polls the user has access to from the backend
+//TODO Need a way to know which polls the users has admin access to and which they do not
+
+// /api/users/
+export default class Groups extends Component {
+  constructor() {
+    super();
+    if (!localStorage.getItem("loggedIn")) {
+      //Redirect("/login");//this is a way to redirect the user to the page
+      //window.location.reload(false);//this forces a reload so this will make the user go to the login page. A little barbaric but it works. If frontend wants to make it better by all means
+    }
+  }
+
+  signout() {
```

Comment:
This is probably not necessary I'd think. I know you were just working off another page so I imagine that's why it's there.

Reply:
Unused Code deleted.